bedrock/foundation/templates/foundation/trademarks/policy.html

```diff
@@ -76,34 +74,33 @@ <h4>{{ _('Don’t:') }}</h4>
 
   <ul class="mzp-u-list-styled">
 
-    <li>{{ _('Don’t use Mozilla trademarks unless specifically permitted under these General Guidelines or the Additional Guidelines below.') }}</li>
-
     <li>{{ _('Don’t use Mozilla trademarks in the name of your business, product, service, app, domain name, publication, or other offering.') }}</li>
 
-    <li>{{ _('Don’t use marks, logos, slogans, domain names, or designs that are confusingly similar to Mozilla trademarks.') }}</li>
-
-    <li>{{ _('Don’t use Mozilla trademarks in a way that implies affiliation with, or sponsorship, endorsement, or approval by Mozilla of your products or services.') }}</li>
+    <li>{{ _('Don’t use marks, logos, company names, slogans, domain names, or designs that are confusingly similar to Mozilla trademarks.') }}</li>
 
-    <li>{{ _('Don’t display Mozilla trademarks more prominently than your product or service name.') }}</li>
+    <li>{{ _('Don’t use Mozilla trademarks in a way that incorrectly implies affiliation with, or sponsorship, endorsement, or approval by Mozilla of your products or services.') }}</li>
 
-    <li>{{ _('Don’t use Mozilla trademarks on merchandise for sale (e.g., selling t-shirts, mugs, etc.)
-or other commercial use.') }}</li>
+    <li>{{ _('Don’t display Mozilla trademarks more prominently than your product, service, or company name.') }}</li>
 
-    <li>{{ _('Don’t modify Mozilla’s trademarks, combine them with any other symbols, words, or images, or incorporate them into a tagline or slogan.') }}</li>
+    <li>{{ _('Don’t use Mozilla trademarks on merchandise for sale (e.g., selling t-shirts, mugs, etc.)') }}</li>
 
-    <li>{{ _('Don’t use Mozilla trademarks in a way that is misleading, unfair, defamatory, libelous, disparaging, obscene, or sullies Mozilla’s reputation.') }}</li>
+    <li>{{ _('Don’t use Mozilla trademarks for any other form of commercial use (e.g. offering technical support services), unless such use is limited to a truthful and descriptive reference (e.g. “Independent technical support for Mozilla’s Firefox browser”).') }}</li>
 
-    <li>{{ _('Don’t use Mozilla trademarks in association with any illegal activities or materials.') }}</li>
+    <li>{{ _('Don’t modify Mozilla’s trademarks, abbreviate them, or combine them with any other symbols, words, or images, or incorporate them into a tagline or slogan.') }}</li>
 
   </ul>
 
   <h3>{{ _('Additional Guidelines') }}</h3>
 
+  <h4>{{ _('Software Distribution Guidelines') }}</h4>
+
+  <p>{% trans distribution=url('foundation.trademarks.distribution-policy') %}<a href="{{ distribution }}">Click here</a> for additional guidelines that apply to distribution of the Firefox Browser and other Mozilla software.{% endtrans %} </p>
```

Comment:
Not a blocker: reword "click here" link?

Reply:
Asked for different copy for this section.